[Internal] Updated Release Procedure (#1113)
TTT-2/TTT2: RELEASE_PROCESS.md

        - Update the `GM.Version` string to the new version.
        - Check if other `GM` strings are up to date as well.

>     1. Clean up the language files with out leanguage cleanup tool

Reviewer: ```suggestion
    1. Clean up the language files with our leanguage cleanup tool
```

>     1. Clean up the language files with our leanguage cleanup tool

Reviewer 2: Just noticed a spelling mistake I think? `leanguage`

        - Tool: https://github.com/TTT-2/ttt2-language_parser
        - Run from folder as: `python parse.py --in "../TTT2/lua/terrortown/lang" --out "../TTT2/lua/terrortown/lang" --base "en"`

>         - Makes sure all language files fit to the english translation

Reviewer: ```suggestion
        - Makes sure all language files align with the english translation
```
